README.md

```diff
@@ -1,86 +1,99 @@
 # IAM-CTL
 
-This will use to get support for create service providers inside the Identity Server.
-Here you can create service providers by entering inputs as a command  and flags or entering inputs in an interactive way.
-
+The IAM-CTL is a tool that can be used to manage WSO2 Identity Server configurations from the command line. It is written in Go and uses the WSO2 Identity Server management REST APIs to manage configurations.
 
 ### Pre-Requisites
-* WSO2IS 5.10.0 alpha2 and start the server.
+You need to [setup](https://is.docs.wso2.com/en/6.0.0/get-started/sample-use-cases/set-up/) the WSO2 Identity Server 6.1.0.
 
-### How to run the executable file 
+### Run the tool
 
-1. Download latest release binary file from [Releases](https://github.com/wso2-extensions/identity-tools-cli/releases)
+1. Download the latest release binary file from [Releases](https://github.com/wso2-extensions/identity-tools-cli/releases)
  based on your Operating System.
-2. Extract the tar or zip file.
-* linux:
 
-```<iamct-{version}>/bin/iamctl```
+2. Extract the `tar` or `zip` file.
 
-* mac:
+    Here onwards, the extracted directory path is referred to as `<IAM-CTL-PATH>`.
 
-```<iamct-{version}>/bin/iamctl```
+3. Open a terminal and create an alias for the `IAM-CTL` executable file using the following command according to your platform.
+   * linux/mac:
+ 
+       ```
+       alias iamctl="<IAM-CTL-PATH>/bin/iamctl" 
+       ```
 
-* windows
+   * windows
 
-```<iamct-{version}>/bin/iamctl.exe```
+       ```
+       doskey iamctl=<IAM-CTL-PATH>/bin/iamctl.exe $*
+       ```
+ 
+4. Run the tool using the following command to get the basic details.
+    ```
+    iamctl -h
+    ```
+5. Start the IS server and [create an application](https://is.docs.wso2.com/en/6.1.0/guides/applications/register-sp) with **Management Application** enabled.
+6. Configure Oauth inbound authentication configuration with a dummy callback URL and make note of the client ID and client secret.
 
-Eg: If it is mac and the released version is 1.0.0, then the extracted binary file looks like ```iamct-1.0.0/bin/iamctl```
 
+The IAM CTL can be used in two basic modes.
+## CLI Mode
 
-3. Then you select your directory to work and open a terminal. Here onwards refer the location of ```<iamct-{version}> ``` directory as ```<IDENTITY-CLI-PATH>```
-4. You can choose any name as keyword of IAM-CTL and set that using following command according to your platform.
-Here I have chosen 'iamctl' as keyword.
+The CLI mode can be used to handle bulk configurations in the target environment. This can be used to propagate resources across multiple environments, deploy new configurations to target environments and will act as a backup of each environment's configurations.
```

Comment:
```suggestion
The CLI mode can be used to handle bulk configurations in the target environment. This can be used to propagate resources across multiple environments, deploy new configurations to target environments, and act as a backup of each environment's configurations.
```